network/src/commonMain/kotlin/com/wire/kalium/network/api/contact/search/ContactSearchRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.network.api.contact.search
+
+import kotlinx.serialization.SerialName
+
+data class ContactSearchRequest(
+    @SerialName("q")
+    val searchQuery: String,
+    @SerialName("domain")
+    val domain: String
+    @SerialName("size")
+    val resultSize: Int? = null
+)
```

Comment:
```suggestion
data class ContactSearchRequest(
    val searchQuery: String,
    val domain: String
    val resultSize: Int? = null
)
```
No need for the SerialName since it won't be serialized

Reply:
true 🤦